src/schema/v2/conversation/index.ts

```diff
@@ -303,10 +304,41 @@ export const ConversationType = new GraphQLObjectType<any, ResolverContext>({
         }
       },
     },
+    collectorResume: {
+      description:
+        "The collector profile of the user who initiated the conversation. Do not use this field for Partners",
+      type: CollectorResume,
+      resolve: async (
+        { from_id, to_id },
+        _args,
+        { partnerCollectorProfileLoader }
+      ) => {
+        if (!partnerCollectorProfileLoader) return
+
+        try {
+          const data = await partnerCollectorProfileLoader({
+            partnerId: to_id,
+            userId: from_id,
+          })
+
+          return {
+            collector_profile: data.collector_profile,
+            follows_profile: data.follows_profile,
```

Comment:
`followsProfile` vs `hasPartnerFollow` - we might want to consolidate this naming and avoid confusion downstream